source/patterns/01-molecules/figure/figure~ar-caption.yaml

```diff
@@ -7,8 +7,11 @@ content:
     image:
       src: https://picsum.photos/600/400
       alt: ''
+      class: 'figure__image'
```

Comment:
Not needed as the equivalent is now provided by https://github.com/libero/pattern-library/pull/87/files#diff-0f2e58fc0cbc0ea5bd688774e41a1a43R9.